code/game/jobs/job/supervisor.dm

```diff
@@ -151,16 +151,17 @@ GLOBAL_DATUM_INIT(captain_announcement, /datum/announcement/minor, new(do_newsca
 /datum/job/blueshield
 	title = "Blueshield"
 	flag = JOB_BLUESHIELD
-	department_flag = JOBCAT_KARMA
+	department_flag = JOBCAT_ENGSEC
 	total_positions = 1
 	spawn_positions = 1
 	supervisors = "the Nanotrasen representative"
 	department_head = list("Captain")
 	selection_color = "#ddddff"
-	req_admin_notify = 1
-	is_command = 1
+	req_admin_notify = TRUE
+	is_command = TRUE
 	transfer_allowed = FALSE
 	minimal_player_age = 21
+	exp_map = list(EXP_TYPE_COMMAND = 3000) // 50 hours baby
```

Comment:
I am open to suggestions on whether this should be just command or a composite of other depts

Reply:
Could be a combination of primarily sec hours with some command and medical hours. You're essentially a bodyguard and medic for command after all

Reply:
I want command hours to be the primary part. High security hours will encourage large amounts of redshielding.

Reply:
Fair point. Maybe Something similar to what GDNgit said below though maybe 40 command hours, so you know the hierarchy, 15-20 medical hours so you know basic healing, and 20 sec hours so you know some combat skills

Reply:
You don't need 15-20 medical hours to know 'basic healing'. That's the equivalent of 10 rounds playing as medical. You need way less than that to know the basics of healing. Unless your one of those Blueshields who shoves doctors/surgeons out of the way because you just happen to know how to do surgery. That really isn't your job. Doing surgery in a pinch because there's no doctors about as a result of some calamity is fine, but it is the exception not the rule. Forcing 15+ hour medical on what is essentially a bodyguard role is way overboard and just like redshield, you are encouraging medshields. I would not be surprised if we get Blues building their own medical bay on the bridge next. Isn't this the reason why brig physician got removed? It deprived medical of playing a part. I see no difference here.

Blueshields should know a little bit of everything; but the suggestion above and by GDNgit go way beyond what a bodyguard role should be. 50+ security job will absolutely lead to more redshields and will do nothing meaningful for robustness. Keeping the Blueshield confined to command hours seems reasonable to me, and no more than 2-4 hours in medical if need be, but even that I feel is not required. Blueshields shouldn't be doctorate surgeons as well just because it makes you more of a 'robust gamer'.

Reply:
2-4 hours can be one to two rounds in medbay. I personally disagree with there even being a bar if it's that low.
I think 10 hours of medbay is a fair play, I feel like by the time you get around to playing blueshield you should have at least tried medbay out enough to get a feeling for the common maladies (like how important IB is, dealing with a broken bone, or keeping people from dying). I agree that blueshields shouldn't need to be surgeons, but if we're gonna require medbay hours they should at least have the know-how to be a paramedic-lite.

Reply:
Bit late at night, but I'm going to try to refute your points one by one

>You don't need 15-20 medical hours to know 'basic healing'.

It's common for new doctors to struggle with properly healing their patients, even into the 40-50 hour range, and it's totally unreasonable to expect otherwise as the vast majority of the playerbase are made of casual players that only hop on occasionally. Not everyone is going to have time to read the wiki or learn what specific chems do. However, this isn't my main reason, I'll get to that soon.

>"the entire medshield rant"
 (I'm not posting that here I'm on mobile)
So, this really does feel like a problem related to your own personal experience. I have seen 1 player which could be considered a medshield here, and they eventually got the memo and left. I would also like clarification on what a medshield would actually be, as it's not exactly clear.

>"bluebay"

This is a bit of a red flag to me that you might be doing this in poor faith, for multiple reasons a blueshield will not realistically build a small medbay mostly due to lacking the pre setup in the brigbay.

>entire second paragraph 

The main reason I do not want to see blueshield have a... 4-5 hour medical requirement is because it partly defeats another purpose of time locking, encouraging your to try other deparments. 1-2 rounds of med will teach you jack shit, very simply put and will not encourage you to actually stick around with the deparment. If a timelock is going to be there SIMPLY to be there then cut that shit. I feel I already adequately addressed why sec/med hours are needed anyways.
I also cannot in good faith encourage players with relatively limited experience to play blueshield, if it doesn't end with them being harassed for their actions I would be honestly shocked. 

Ps: I hope this is not the case, but your phrasing made this seem like it was supposed to directed at me. Maybe just mind it a bit.